.setup/bin/setup_sample_courses.py

```diff
@@ -208,12 +209,8 @@ def main():
     os.system("rm /tmp/hwcron_cron_backup.txt")
     os.system("systemctl restart submitty_grading_scheduler")
 
-def generate_random_user_id(length):
-    id_base = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
-    anon_id = ""
-    for i in range(length):
-        anon_id += random.choice(id_base)
-    return anon_id
+def generate_random_user_id(length=15):
+    return ''.join(random.choice(string.ascii_lowercase + string.ascii_uppercase +string.digits) for _ in range(length))
```

Comment:
you can replace `string.ascii_lowercase + string.ascii_uppercase` with just `string. ascii_letters`